akka-docs/src/main/paradox/stream/operators/Source/asSubscriber.md

```diff
@@ -1,6 +1,6 @@
 # asSubscriber
 
-Integration with Reactive Streams, materializes into a `org.reactivestreams.Subscriber`.
+Integration with Reactive Streams, materializes into a @javadoc[Subscriber](java.util.concurrent.Flow.Subscriber).
```

Comment:
Somewhat confusing with the `Source.asSubscriber`/`operators/Source/asSubscriber` and then documenting the `JavaFlow.asSubscriber` but I guess it's hard to make it completely non-confusing...